app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (150, 38, [33, 4]),
+        (-5, -10, [0, 0]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 28, [2, 2]),
+        (28, 29, [3, 3]),
```

Comment:
There is a repetition in testing similar age values for both cat and dog. According to the checklist, make sure that your tests don't test the same case multiple times. Consider removing or modifying redundant test cases.